kfc/kfc-gradle-plugin/src/main/kotlin/com/github/turansky/kfc/gradle/plugin/PatchWebpackConfig.kt

```diff
@@ -108,10 +123,40 @@ open class PatchWebpackConfig : DefaultTask() {
             .asFile
 }
 
+@Suppress("JSUnnecessarySemicolon")
+private fun createReplacePatch(replacements: Map<String, Tuple2<String, Boolean>>): String? {
+    if (replacements.isEmpty())
+        return null
+
+    val replacementOptions = replacements.map { (oldValue, entry) ->
```

Comment:
`map` can be merged with `join`

Reply:
As it is a map, we can do so after transforming it to a `collection`. I don't think merging `map` and` join` adds much in terms of clarity nor will such a merge have a meaningful impact on performance.

Reply:
OK